ciphers/morse_code.py

```diff
@@ -29,15 +71,15 @@ def encrypt(message: str) -> str:
     >>> encrypt("SOS!") == encrypt("sos!")
     True
     """
-    return " ".join(MORSE_CODE_DICT[char] for char in message.upper())
+    return " ".join(MORSE_CODE_DICT.get(char.upper(), "") for char in message)
```

Comment:
The proposed change is less efficient because it calls two functions for every letter instead of calling one function once.